### PR TITLE
Add internal application for adding units to the DB

### LINKED ISSUE #338: Update site center coordinates in DB (adding EPSG, adding units)
Site center parameters are stored in the db without units (e.g. altitude or lon/lat).

Also good to add EPSG, for easier reference:   North is EPSG=32628, for the South it is 32719

### PR DESCRIPTION
**Pull request by orelgueta**
The application added in this PR should not really be used so I did not write much documentation for it nor added it to the tests. We should probably come up with a solution for how to deal with such applications, maybe put them in a separate directory. Fixes #338.

### CODE REVIEW

File: applications/add_unit_to_parameter_in_db.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python3
+
+"""
+    Summary
+    -------
+    This application is used to add a unit to parameters in the DB.
+
+    This application should not be used by anyone but expert users and only in extreme cases.
```
Comment (GernotMaier) [inline applications/add_unit_to_parameter_in_db.py:8]: Suggest to replace `extreme` by `unusual`.

File: applications/add_unit_to_parameter_in_db.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    Therefore, no additional documentation about this applications will be given.
```
Comment (GernotMaier) [inline applications/add_unit_to_parameter_in_db.py:9]: I don't think this is the right thing to do - safety by not giving documentation? Doesn't this require write access to the DB, which most users don't have?

File: applications/add_unit_to_parameter_in_db.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    cfg.setConfigFileName(args.configFile)
+
+    logger = logging.getLogger()
+    logger.setLevel(gen.getLogLevelFromUser(args.logLevel))
+
+    # parsToUpdate = ["altitude"]
+    parsToUpdate = ["ref_long", "ref_lat"]
+
+    # units = ["m"]
+    units = ["deg", "deg"]
```
Comment (GernotMaier) [inline applications/add_unit_to_parameter_in_db.py:36]: Oh, now I understand. The changes are hardwired? So you hardly ever expect to this be used?

File: simtools/db_handler.py [hunk 4]
```diff
@@ -1262,3 +1283,60 @@ def insertFilesToDB(self, filesToAddToDB, dbName=DB_CTA_SIMULATION_MODEL):
...
+            _telModelNameValidated = names.validateTelescopeModelName(telescopeModelName)
+            _telNameDB = self._getTelescopeModelNameForDB(_siteValidated, _telModelNameValidated)
+            query["Telescope"] = _telNameDB
+        elif collectionName == "sites":
+            query["Site"] = _siteValidated
+        else:
+            raise ValueError("Can only get versions of the telescopes and sites collections.")
+
+        _allVersions = list()
+        for post in collection.find(query):
```
Comment (GernotMaier) [inline simtools/db_handler.py:1336]: Ha! Suggest to replace the non-pythonic for loop by list comprehension...:-)

**Review by GernotMaier — APPROVED** [review]
Minor comment about the application and documentation.

File: applications/add_unit_to_parameter_in_db.py (hunk 1 above)
Comment (orelgueta) [reply applications/add_unit_to_parameter_in_db.py]: Done

File: applications/add_unit_to_parameter_in_db.py (hunk 2 above)
Comment (orelgueta) [reply applications/add_unit_to_parameter_in_db.py]: As discussed, moved this application to a sub directory called `db_development_tools` for specific tools like that which do not require documentation.

Technically the API user can make these changes as well. We also have a read-only user but we do not use it (maybe we should start).

File: simtools/db_handler.py (hunk 4 above)
Comment (orelgueta) [reply simtools/db_handler.py]: When you're right, you're right!

**Comment by orelgueta** [comment]
Merging, failure in integration test is solved in #344

**Pull request merged**